<ui version="4.0" >
 <class>CreateListGroup</class>
 <widget class="QDialog" name="CreateListGroup" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>0</hsizetype>
    <vsizetype>0</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Selection of groups</string>
  </property>
  <property name="autoFillBackground" >
   <bool>true</bool>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="GBButtons" >
     <property name="title" >
      <string/>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="3" >
       <widget class="QPushButton" name="buttonHelp" >
        <property name="text" >
         <string>Help</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QPushButton" name="buttonCancel" >
        <property name="text" >
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QPushButton" name="buttonApply" >
        <property name="text" >
         <string>Apply</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QPushButton" name="buttonOk" >
        <property name="text" >
         <string>OK</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="GBOptions" >
     <property name="title" >
      <string>Selected groups</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <widget class="QTableWidget" name="TWGroupe" >
        <property name="showGrid" >
         <bool>true</bool>
        </property>
        <property name="rowCount" >
         <number>0</number>
        </property>
        <property name="columnCount" >
         <number>2</number>
        </property>
        <column>
         <property name="text" >
          <string>Selection</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>Group</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
